bench-vortex/src/bin/clickbench.rs

```diff
@@ -41,15 +51,13 @@ struct Args {
     #[arg(short, long, value_delimiter = ',')]
     queries: Option<Vec<usize>>,
     #[arg(long, default_value_t = false)]
-    emit_plan: bool,
-    #[arg(long, default_value_t = false)]
     emulate_object_store: bool,
     #[arg(long)]
     export_spans: bool,
     #[arg(long, value_enum, default_value_t = Flavor::Partitioned)]
     flavor: Flavor,
     #[arg(long)]
-    use_remote_data_dir: Option<String>,
+    remote_data_dir: Option<String>,
```

Comment:
should we get the old flag name to work as well? Would be cool to keep the old s3 run commands to work. At least we should change the ci bench runners to use the new flag